types/altv-client.d.ts

```diff
@@ -463,6 +469,11 @@ declare module "alt-client" {
   }
 
   export class Discord {
+    /*
+     * Returns the current Discord user. Can be null.
```

Comment:
In its current shape this comment is obsolete, because it brings nothing new beyond what you can find out looking at the name of the property itself